src/dotnet-svcutil/lib/NamedPipeMetadataImporter/IMetadataExchange.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+
+namespace Microsoft.Tools.ServiceModel.Svcutil
+{
+    [ServiceContract]
+    public interface IMetadataExchange
```

Comment:
How does svcutil tool handle this?  Is there a similar class in the svcutil tool?

Reply:
This is okay to duplicate here.